bigbluebutton-html5/package.json

```diff
@@ -20,6 +20,12 @@
   },
   "dependencies": {
     "babel-runtime": "~6.26.0",
+    "//": [
+      "core-js is included with babel-runtime",
+      "but Meteor 1.6.0.1 doesn't see it there for some reason",
```

Comment:
Did you want to open an issue for that if you're still investigating it?

Reply:
I investigated.. a guy who had Meteor installed on Windows 10 had the same problem, and he fixed it by reinstalling Meteor. :-)
I haven't tried reinstalling Meteor, I will do it now!

I also wanted to comment this line, since this package is not imported/used directly, and it can be most likely removed in the future updates.